### DISCUSSION TITLE
Sometimes you just have to laugh

### DISCUSSION THREAD

**Discussion by gardnmi** [CMSgov/price-transparency-guide#731] [General]
![Capture](https://github.com/CMSgov/price-transparency-guide/assets/7167260/5dcb6b77-95cb-4530-9b8e-d94ed6484d75)

Source:
https://individual.carefirst.com/individuals-families/mandates-policies/machine-readable-file.page?submit=true&componentID=1645309246851

**Comment by JR-CMS** [comment]
Thank you for bringing this to our attention. We will review the information you provided. If you believe you are observing instances of non-compliance in the future, please submit it to us at https://www.cms.gov/healthplan-price-transparency/contact-us.
